lib/staging/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_subdirectory(can_dpm1000)
+add_subdirectory(external_energy_limits)
```

Comment:
@hikinggrass probably we can include this only when either API, OCPP or OCPP201 are build but Im not quite sure how. Can you help with that?